simln-lib/src/lib.rs

```diff
@@ -1559,4 +1593,39 @@ mod tests {
             payment_interval
         );
     }
+
+    #[tokio::test]
+    async fn test_validate_zero_amount_no_valid() {
+        let mut node_1 = test_utils::create_nodes(1, 100_000)
+            .first()
+            .unwrap()
+            .0
+            .clone();
+        node_1.features.set_keysend_optional();
+        let mut node_2 = test_utils::create_nodes(1, 0).first().unwrap().0.clone();
+        node_2.features.set_keysend_optional();
+
+        let mock_node_1 = MockLightningNode::new();
+        let mock_node_2 = MockLightningNode::new();
+        let mut clients: HashMap<
+            bitcoin::secp256k1::PublicKey,
```

Comment:
nit: import with `use`, same for `SimulationConfig` below - generally have always been importing like this in the project rather than inlining